src/mount/readdata_cache.h

```diff
@@ -381,11 +382,12 @@ class ReadCache {
 		return entries_.insert(it, *e);
 	}
 
-	void collectGarbage(unsigned count = 4) {
+	void collectGarbage(uint32_t usedReadCacheMemoryMb, unsigned count = 4) {
```

Comment:
Same comment as before here, were you not trying to update the value of gUsedReadCacheMemory through the changes of usedReadCacheSize in query function which comes down here?

Also, is the value stored in usedReadCacheMemoryMb representing megabytes? Because there is a substraction a bit down here which is substracting bytes (the buffer size).

Reply:
Done